docs/general-concepts/forms-fields/standard-fields/calendar.md

```diff
@@ -30,6 +30,8 @@ the text box. Otherwise the default value, if any, is displayed.
 - **filltable** (optional): if set to true, dates of the previous and next month are added at the top and bottom of the current month to fill the grid. Defaults to true.
 - **minyear** (optional): set a signed integer for a number of years (-10, -2, 0, 7, 12, ...) to define the relative lower limit for the year selection. The user cannot select a year before this limit. If zero, which is the default, there is no limit.
 - **maxyear** (optional): set a signed integer for a number of years (-10, -2, 0, 7, 12, ...) to define the relative upper limit for the year selection. The user cannot select a year after this limit. If zero, which is the default, there is no limit.
+- **default** (optional): a default date can be specified in `Y-m-d` format. You can default to the current date by using `default="now"` as the xml attribute.
+  You can also directly set the date to the current day by typing the word "now" into the rendered text box instead of pressing the "Today" button
```

Comment:
I'm not sure what you mean with that? in which text box? as user?

Reply:
yep, the user text box rendered by the control. but by making the same change as you suggested in the other version it is clearer still and the last sentence is not relevant